cypress/integration/test.js

```diff
@@ -103,6 +113,10 @@ describe('MovieDetail', () => {
   it('Should have a rating', () => {
     cy.get('p').should('contain', 'Freshness:');
   });
+
+  it('Should contain trailers and/or clips', () => {
+    cy.get('article').should('have.class', 'trailer-container')
```

Comment:
This is coming up with the trailers for Mulan because of the url we're testing. We need to `intercept` the fetch request that's getting the videos and send links to some videos that aren't dependent on an actual server. It's hurting my head a little bit to think of dummy data for this so I'd be into collaborating on figuring it out if you want. I feel like we'll have to do some digging to get the right object.

Reply:
Added an intercept and added dummy data for the video testing.